configs/nvme-storage-configurator/base/configure.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Enable debugging
+set -eux
+
+if [ -f /host-etc-systemd-dir/nvme-init-done ]; then
+    echo 'NVMe init already done'
+    exit 0
+fi
+
+cp -f /scripts/configuration.sh /host-usr-local-bin/configuration.sh
```

Comment:
I find the names of the scripts confusing (maybe because they similar).
Lets give them meaningful names.

we can call the script that install the unit file and the script it runs `install.sh`
and the the script that does the mount `mount.sh`

Reply:
ok lets change it. I will change them in next commit.

Reply:
I have changed the secripts to `install.sh `and `mount.sh`.